common/src/main/java/net/draycia/carbon/common/listeners/RadiusListener.java

```diff
@@ -74,6 +76,10 @@ public RadiusListener(
                     return false;
                 });
             }
+
+            if (!event.recipients().isEmpty()) return;
```

Comment:
In general, I think our code style calls for always including brackets, not sure why it's not in the checkstyle. I think in this case
```java
if (condition) {
  // action...
}
```
would be better anyways though.